2020-02-Spring/1_strings/reverse_string/Driver.java

```diff
@@ -0,0 +1,44 @@
+public class Driver{
+    public static void main (String []args){
+        char [] c1 = {'h'};
+        char [] c2 = {'h', 'e'};
+        char [] c3 = {'h', 'e', 'l','l','o'};
+        char [] c4 = {'c','a','w'};
+        char [] c5 = {'r','a','c','e','c','a','r'};
+        char [] c6 = {'1','2','3','4'};
+
+        System.out.println("Output: ");
+        reverseString(c1);
```

Comment:
Feel free to ignore this suggestion, but maybe add the expected output next to each test so it's easy for someone using the driver to check what the expected output to be?

Reply:
Made a change, a very good suggestion. Thanks!